cosmo_tester/resources/terraform/scripts/linux-cli-test.sh

```diff
@@ -24,20 +20,18 @@ fi
 
 set -e
 
-echo "Creating inputs file.."
-echo "public_ip: ${PUBLIC_IP}
-private_ip: ${PRIVATE_IP}
-ssh_user: ${MANAGER_USER}
-ssh_key_filename: ${PRIVATE_KEY_PATH}
-admin_username: admin
-admin_password: admin" > ${INPUTS_FILE_PATH}
-cat ${INPUTS_FILE_PATH}
+cfy profiles use ${PRIVATE_IP} -u admin -p admin -t default_tenant
+
+cfy blueprints upload https://github.com/cloudify-cosmo/cloudify-hello-world-example/archive/master.tar.gz -b bp -n singlehost-blueprint.yaml
+cfy deployments create -b bp dep -i server_ip=${PRIVATE_IP} -i agent_user=${MANAGER_USER} -i agent_private_key_path=${AGENT_KEY_PATH}
+cfy executions start install -d dep
 
-echo "Setting permissions for private key file: ${PRIVATE_KEY_PATH}"
-chmod 400 ${PRIVATE_KEY_PATH}
+echo "Validating hello world is working..."
+# Because of set -e, this will fail if the string isn't found in the output
```

Comment:
As we're depending on that, the set -e should probably be commented to indicate that it's critical to the tests.